app/src/main/java/com/example/ohthmhyh/CERecycleviewAdapter.java

```diff
@@ -191,4 +194,48 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
      */
         void onItemclicked(int position);
     }
+    /**
+     *This method is used to open a conformation screen with the user before a delete
+     * @param position the position of the item being swiped
+     * The reason why it is like this is because dialogs are asynchronous so if the delete method is outside
+     * it will be ran before the user input, this way makes it so the user input does something
+     */
+    public void openDiolog(int position){
```

Comment:
It's actually spelt 'Dialog'. Android Studio has a feature where if it detects a word is spelt incorrectly, then it underlines it. Please don't hesitate to use that feature.

I'll make the change here though.